### ISSUE TITLE
Change Server/Database dynamically

### ISSUE THREAD

**Issue by ljkern** [mhertzfeld/CrystalReportsNinja#4]
Is it possible With a SQL Server connection to either use the -S option and change the ODBC DSN at runtime or use the -D and use a different database at runtime? I need to run the same crystal report against different databases. (They all have the same structure). It seems no matter what I do, it uses what is defined in the Crystal Report.  Any ideas how to make this work?

**Comment by mhertzfeld** [comment]
Just saw your post.  I'll take a look and let you know what I find.

**Comment by mhertzfeld** [comment]
Seems to be working for me.  I specified the DSN name for both -D and -S and it worked as expected.

I build a demo report to query table A in database ZZ.  I copied table A to database YY and was able to use the -D and -S option to specify the DSN of database YY and ran ninja.  It produced the results I expected.

DSNs are sensitive to 32bit vs 64bit so make sure that if your running 64bit ninja you have 64bit DSNs configured.  Same goes for 32bit.

Could be something strange in the report itself that's causing the problem or the enviroment.  If you haven't done so already, try creating a new very simplistic report and try testing with that.
